corehq/apps/toggle_ui/views.py

```diff
@@ -194,6 +197,26 @@ def post(self, request, *args, **kwargs):
         toggle.save()
 
         changed_entries = previously_enabled ^ currently_enabled  # ^ means XOR
+
+        added_entries = currently_enabled - previously_enabled
+        is_deprecated_toggle = (self.static_toggle.tag == TAG_DEPRECATED)
+        if added_entries and (self.static_toggle.notification_emails or is_deprecated_toggle):
+            subject = "User {} added {} on {} in environment {}".format(
+                self.request.user.username, self.static_toggle.slug,
+                added_entries, settings.SERVER_ENVIRONMENT
+            )
+
+            if self.static_toggle.notification_emails:
+                emails = [
+                    "{}@{}.com".format(email, "dimagi")
+                    for email in self.static_toggle.notification_emails
+                ]
+                _assert = soft_assert(to=emails, send_to_ops=is_deprecated_toggle)
+            else:
+                _assert = soft_assert(send_to_ops=is_deprecated_toggle)
+
+            _assert(False, subject)
```

Comment:
You could stick this stuff in a private method to keep the `post` method relatively small.

Reply:
https://github.com/dimagi/commcare-hq/pull/21273/commits/4a56ebeba9e34b390f4a88e6c71641fa6aec98c7